examples/gno.land/r/nt/boards2/v1/freeze.gno

```diff
@@ -0,0 +1,76 @@
+package boards2
+
+import "std"
+
+type freezeArgs struct {
+	boardID  BoardID
+	threadID PostID
+	replyID  PostID
+}
```

Comment:
The type could be removed if arguments are used in `setThreadReadOnly` and `setReplyReadOnly` functions instead of passing a type. If so, that change might simplify the code